linkcheck/director/aggregator.py

```diff
@@ -18,7 +18,10 @@
 Aggregate needed object instances for checker threads.
 """
 import threading
-import thread
+try: # Python 3
+    import _thread
+except ImportError:
+    import thread as _thread
```

Comment:
here's another example. above we do `from thread import error as thread_error` yet here we do `import thread as _thread`... it looks inconsistent - why don't we do the latter everywhere?